modules/phase_field/src/materials/InterfaceOrientationMultiphaseMaterial.C

```diff
@@ -85,52 +84,58 @@ InterfaceOrientationMultiphaseMaterial::computeQpProperties()
   if (n < -cutoff)
     n = -cutoff;
 
+  // Calculate the orientation angle
   const Real angle = std::acos(n) * MathUtils::sign(ny);
```

Comment:
this should be rewritten using `atan2` (but you don't have to do that in this PR)